alf/utils/schedulers.py

```diff
@@ -29,6 +31,7 @@ class Scheduler(object):
     * "percent": percent of training completed.
     * "iterations": the number training iterations.
     * "env_steps": the number of environment steps
+    * "global_counter": the value from ``alf.summary.get_global_counter()``
```

Comment:
Is the potential difference between "iterations" and "global_counter" only due to the flag "update_counter_every_mini_batch"?

Reply:
For supervised learning,  iteration means epoch. So global_counter is increased many time in train_iter.